framework/src/base/MooseApp.C

```diff
@@ -321,6 +321,9 @@ MooseApp::validParams()
   params.addParam<bool>(
       "automatic_automatic_scaling", false, "Whether to turn on automatic scaling by default.");
 
+  params.addCommandLineParam<std::string>(
```

Comment:
The type `MooseEnum` is supported for command line parameters now. You should set your enum here so you get a better error when a non-valid choice is provided.

Reply:
noice, thx